src/components/Auth.vue

```diff
@@ -279,7 +295,7 @@ export default {
       axios.get(process.env.FIREROAD_URL + '/fetch_token/?code=' + code)
         .then(function (response) {
           if (response.data.success) {
-            if (this.data.authCookiesAllowed) {
+            if (this.data.cookiesAllowed) {
               this.data.$cookies.set('accessInfo', response.data.access_info);
```

Comment:
@georgiashay is `this.data.$cookies` right? It's weird to me that there's a `$` in there

Reply:
Yeah, the $cookies is correct.  It's how the vue-cookies library works.